tests/test_lab_train.py

```diff
@@ -321,6 +318,11 @@ def test_train_linux(
         linux_train_mock,
         is_macos_with_m_chip_mock,
     ):
+        # pylint: disable=consider-using-with
```

Comment:
nit: you may be able to avoid arguing with the linter about this if you declare a helper function instead of using a `lambda`?

Reply:
thanks, the lambda has been replaced